### DISCUSSION TITLE
Conflicts with the multiple NPM packages

### DISCUSSION THREAD

**Discussion by abhishekpnt** [Sunbird-inQuiry/Community#47] [General Discussions]
@surendrasinghs @pallakartheekreddy 

For the dependency [@project-sunbird/sunbird-quml-player-v9](https://www.npmjs.com/package/@project-sunbird/sunbird-quml-player-v9) 5.1.5 is the latest version and we are currently using it  in the portal.
But i see one more npm package with the name [@project-sunbird/sunbird-quml-player](https://www.npmjs.com/package/@project-sunbird/sunbird-quml-player) which is last updated 5 days ago.

Also for the dependency [@project-sunbird/sunbird-collection-editor-v9](https://www.npmjs.com/package/@project-sunbird/sunbird-collection-editor-v9) 5.0.17 is the latest version . But i see one more NPM package with the name [@project-sunbird/sunbird-collection-editor](https://www.npmjs.com/package/@project-sunbird/sunbird-collection-editor)

Can you please let us know which is the latest dependency we have to use in Sunbird-Ed portal for the next angular migration from 11 to 12? 

----

CC: @Krishnaj20 @sharathkashyap @rajeevsathish @vinukumar-vs

**Comment by pallakartheekreddy** [comment]
@abhishekpnt Please check below release notes. We have added a **Deprecation Warning** in **Summary of the changes** as part of release-5.1.0, [@project-sunbird/sunbird-collection-editor-v9](https://www.npmjs.com/package/@project-sunbird/sunbird-collection-editor-v9) and [@project-sunbird/sunbird-quml-player-v9](https://www.npmjs.com/package/@project-sunbird/sunbird-quml-player-v9) are deprecated.
https://inquiry.sunbird.org/use/release-notes/inquiry-release-v5.1.0
@Krishnaj20 @itsvick
